src/Components/Blazor/Build/testassets/standalone/standalone.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="$(ReferenceBlazorBuildFromSourceProps)" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RazorLangVersion>3.0</RazorLangVersion>
+  </PropertyGroup>
+
+  <!-- Test Placeholder -->
+
+  <ItemGroup>
+    <!-- We'll only reference a subset of publicly released package versions that are enough to link, build, and publish . -->
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.Mono" Version="3.1.0-preview3.19531.1" />
```

Comment:
Surely we don't want to hardcode any of these version numbers here, do we?

Reply:
The `Blazor.Mono` package can be changed to use the version used by the rest of the repository. I'm not sure about the two Components references. We want to avoid referencing the actual projects both in blazor-wasm and in the 5.0 branch to avoid involving arcade's build requirements in our test projects. 

In RazorSDK's tests, we use a mix of referencing shipped packages for ones that are already shipped or referencing a runtime shim.